requirements.txt

```diff
@@ -1,5 +1,4 @@
 autobahn
-pygame
```

Comment:
We introduced duplication when adding the pipenv configuration. Note to self: open an issue for generating the requirements file from pipenv.